pkg/pillar/cmd/zedrouter/acl.go

```diff
@@ -361,6 +361,14 @@ func aclToRules(aclArgs types.AppNetworkACLArgs, ACLs []types.ACE) (types.IPTabl
 				"-p", "tcp", "--sport", "domain"}
 			aclRule4.Action = []string{"-j", "ACCEPT"}
 			rulesList = append(rulesList, aclRule1, aclRule2, aclRule3, aclRule4)
+
+			aclRule1.Rule = []string{"-i", aclArgs.BridgeName, "-d", "169.254.169.254",
+				"-p", "tcp", "--dport", "http"}
+			aclRule1.Action = []string{"-j", "ACCEPT"}
+			aclRule2.Rule = []string{"-i", aclArgs.BridgeName, "-s", "169.254.169.254",
```

Comment:
May not be required.

Reply:
I followed the pattern for the tcp domain (aka DNS) rules. Do you think we can clean those up as well?
The DHCP and DNS udp rules are the same as the DNS tcp rules and if we can remove unneeded rules it would be simpler.